guide/daily-spending-wallet/settings.md

```diff
@@ -0,0 +1,213 @@
+    - file: bitcoin-unit
+      modalImage: bitcoin-unit-big
+      alt: Mobile screen with automatic, bitcoin, and satoshi options
+      caption: The bitcoin unit can also be switched contextually in the send screen.
+images_security:
+    - file: security
+      modalImage: security-big
+      alt: Mobile screen with PIN, FaceID, wallet deletion and wallet limit options
+      caption: Options to limit unwanted access and payments.
+    - file: delete-wallet
+      modalImage: delete-wallet-big
+      alt: Confirmation modal for deleting the wallet from the device
+      caption: This screen requires very clear language to ensure users understand the implications.
+images_network:
+    - file: network
+      modalImage: network-big
+      alt: Mobile screen with bitcoin and lightning network, as well as Tor options
+      caption: Overview screen for network options.
+    - file: bitcoin-network
+      modalImage: bitcoin-network-big
+      alt: Mobile screen with bitcoin network customization options
+      caption: Users can choose their own node to connect to.
+    - file: lightning-network
+      modalImage: lightning-network-big
+      alt: Mobile screen with lightning network customization options
+      caption: Information about the users node, and additional security options.
+    - file: watchtower
+      modalImage: watchtower-big
+      alt: Mobile screen with lightning network watchtower options
+      caption: Users can enable watchtower functionality to protect their funds.
+---
+
+{% include picture.html
+    image = "/assets/images/guide/daily-spending-wallet/settings/settings.jpg"
+    retina = "/assets/images/guide/daily-spending-wallet/settings/settings@2x.jpg"
+    mobile = "/assets/images/guide/daily-spending-wallet/settings/settings-mobile.jpg"
+    mobileRetina = "/assets/images/guide/daily-spending-wallet/settings/settings-mobile@2x.jpg"
+    alt-text = "An oil painting of a big, clunky machine with a big bitcoin gear"
+    caption = "How AI thinks Matisse would have visualized bitcoin settings."
+    width = 1600
+    height = 600
+    layout = "full-width"
+%}
+
+<!--
+
+
+This is more of a summary page that bring together all the application settings that
+the user encounters throughout the app experience into a single, concise form.
+
+
+-->
+
+# Settings
+{: .no_toc }
+
+---
+
+<div class="glossary-toc" markdown="1">
+ * Table of contents
+{:toc}
+</div>
+
+---
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/settings/overview.png"
+   retina = "/assets/images/guide/daily-spending-wallet/settings/overview@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/settings/overview-big.png"
+   alt-text = ""
+   width = 250
+   height = 541
+   layout = "float-right-desktop -background -shadow"
+   caption = "The overview screen provides an easy-to-understand grouping of settings."
```

Comment:
Minor nit but having a break line at top and bottom of the list looks less clean then omitting them